src/solvers/dgsem_tree/dg_2d_subcell_limiters.jl

```diff
@@ -12,29 +12,35 @@ function create_cache(mesh::Union{TreeMesh{2}, StructuredMesh{2}, P4estMesh{2}},
                          VolumeIntegralPureLGLFiniteVolume(volume_integral.volume_flux_fv),
                          dg, uEltype)
 
-    A3dp1_x = Array{uEltype, 3}
-    A3dp1_y = Array{uEltype, 3}
     A3d = Array{uEltype, 3}
```

Comment:
We have `A3d` here, but `A3` in the files you changed above. It would be nice to use the same pattern for consistency in all cases. Which version do you prefer?

Reply:
I guess keeping the `d` is closer to the existing code. I will adapt the rest :+1: